### PR TITLE
Update to 3.24 and removed references to WebUniversum

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,50 +14,62 @@
...
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.17.2",
-    "ember-cli-htmlbars": "^4.2.2"
+    "ember-cli-babel": "^7.23.0",
+    "ember-cli-htmlbars": "^5.3.1"
   },
   "peerDependencies": {
     "@lblod/ember-vo-webuniversum": "^0.22.1",
```
Comment (nvdk) [inline package.json:33]: I think it's best to also remove this peer dependency on ember-vo-webuniversum

File: package.json [hunk 2]
```diff
@@ -14,50 +14,62 @@
...
     "@lblod/marawa": "^0.6.0",
```
Comment (nvdk) [inline package.json:34]: perhaps bump marawa to the latest version

File: package.json (hunk 1 above)
Comment (lagartoverde) [reply package.json:33]: Yes, didn't see it, will remove now :)

File: package.json (hunk 2 above)
Comment (nvdk) [reply package.json:33]: I've bumped it in rdfa-editor without issue, but some testing doesn't hurt :)

Comment (lagartoverde) [reply package.json]: I did and tested but not sure that I understand this correctly, it says that the peer should have at least marawa 0.6, shouldn't we try to support a version of marawa as low as possible for compatibility

**Pull request merged**